Source/WebKit/Shared/IPCTester.cpp

```diff
@@ -213,6 +213,12 @@ void IPCTester::syncPing(IPC::Connection&, uint32_t value, CompletionHandler<voi
     completionHandler(value + 1);
 }
 
+void IPCTester::syncPingEmptyReply(IPC::Connection&, uint32_t value, CompletionHandler<void()>&& completionHandler)
+{
+    UNUSED_PARAM(value);
```

Comment:
You can also just omit "value" above.

Reply:
I need value to trigger the deserialization error.